sbe-tool/src/main/java/uk/co/real_logic/sbe/generation/java/JavaGenerator.java

```diff
@@ -3269,7 +3269,14 @@ private void appendDecoderDisplay(
             }
             else
             {
-                append(sb, indent, "builder.append('\\'').append(" + varDataName + "()).append('\\'');");
+                if (characterEncoding.contains("ASCII"))
+                {
+                    append(sb, indent, "builder.append('\\'');");
```

Comment:
@mjpt777 do the single quotes around the string required?
care if i take those out? (both in ASCII and regular implementation)

Reply:
Best to leave them.